test/e2e/dormant-text.cy.ts

```diff
@@ -0,0 +1,129 @@
+import { getIdContent } from "test/support/commands";
+
+context("DormantText Component", () => {
+  describe("Textbox", () => {
+    beforeEach(() => {
+      cy.visit(getIdContent("stage-dormanttext--default"));
+    });
+
+    it("Should type and confirm update in normal width textbox", () => {
+      cy.findByText("Hello there, this is dormanted text").click();
+      cy.findByRole("textbox").clear().type("Updated text normal");
+      cy.findByRole("button").click();
+      cy.findByText("Updated text normal").should("exist");
+    });
+
+    it("Should type and confirm update in full width textbox", () => {
+      cy.findByText(
+        "Hello there, this is dormanted text with full width"
+      ).click();
+      cy.findAllByRole("textbox").clear().type("Updated text full");
+      cy.findAllByRole("button").click();
+      cy.findByText("Updated text full").should("exist");
+    });
+
+    it("Should type and confirm update using Enter key", () => {
+      cy.findByText(
+        "Hello there, this is dormanted text with enable keydown"
+      ).click();
+      cy.findAllByRole("textbox").clear().type("Updated with Enter{enter}");
+      cy.findByText("Updated with Enter").should("exist");
+    });
+  });
+
+  describe("Combobox", () => {
+    beforeEach(() => {
+      cy.visit(getIdContent("stage-dormanttext--with-combobox"));
+    });
+
+    it("Should select option from combobox and accepted by pressing enter", () => {
+      cy.findByText("Apple").click();
+      cy.findByPlaceholderText("Select a fruit...").type(
+        "{downarrow}{downarrow}{enter}"
+      );
+      cy.findByText("Orange").should("exist");
+    });
+
+    it("Should select option from full width combobox", () => {
+      cy.findByText("Banana").click();
+      cy.findByPlaceholderText("Select a fruit full...").type(
+        "{downarrow}{downarrow}{enter}"
+      );
+      cy.findAllByText("Orange").should("have.length.at.least", 1);
+    });
+  });
+
+  describe("Datebox", () => {
+    beforeEach(() => {
+      cy.visit(getIdContent("stage-dormanttext--with-datebox"));
+    });
+
+    it("Should open date picker and click a date", () => {
```

Comment:
let's also test and ensure we have the expected text/date in the dormant text; let's do the same for other examples (ie phonebox, etc) of which the shown/displayed value we haven't assert.